src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -4,10 +4,22 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="property_accessor.cache.prefix" />
+    </parameters>
+
     <services>
         <service id="property_accessor" class="Symfony\Component\PropertyAccess\PropertyAccessor" >
             <argument /> <!-- magicCall, set by the extension -->
             <argument /> <!-- throwExceptionOnInvalidIndex, set by the extension -->
+            <argument /> <!-- cache, set by the extension -->
+        </service>
+
+        <!-- Cache -->
+        <service id="property_accessor.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
And as I said, it should be replaced with filesystem ON by default. And if people want to use APC, they also only need to change ONE line. So this is not a valid reason.